MatrixKit/Controllers/MXKCallViewController.m

```diff
@@ -1174,7 +1190,7 @@ - (void)updateLocalPreviewLayout
     
     CGPoint previewOrigin = self.localPreviewContainerView.frame.origin;
     
-    if (previewOrigin.x != 20)
+    if (previewOrigin.x != (bounds.size.width - _localPreviewContainerViewWidthConstraint.constant - 20.0))
```

Comment:
We can probably define a constant for the "20.0" value in this method.

Reply:
Yeah, updated.